🐛 [BUG]:

### Is there an existing issue for this?

- [X] I have searched the existing issues

### What package does the issue affects?

@nxtensions/astro

### Describe the bug

I can't create a workspace

### Expected behavior

I should be able to create workspacea 

### Steps to reproduce

1. `npx create-nx-workspace@latest --preset=@nxtensions/astro`
2.  Where would you like to create your workspace? · anton
3. Enable distributed caching to make your CI faster · Yes
4. Application name · ttt

```
PS C:\source\test14> npx create-nx-workspace@latest --preset=@nxtensions/astro
(node:40936) Warning: Setting the NODE_TLS_REJECT_UNAUTHORIZED environment variable to '0' makes TLS connections and HTTPS requests insecure by disabling certificate verification.
(Use `node --trace-warnings ...` to show where the warning was created)

 >  NX   Let's create a new workspace [https://nx.dev/getting-started/intro]

√ Where would you like to create your workspace? · anton
√ Enable distributed caching to make your CI faster · Yes

 >  NX   Creating your v16.6.0 workspace.

   To make sure the command works reliably in all environments, and that the preset is applied correctly,
   Nx will run "npm install" several times. Please wait.

✔ Installing dependencies with npm
✔ Successfully created the workspace: anton.
√ Application name · ttt

 >  NX   Cannot find module 'typescript'

   Require stack:
   - C:\source\test14\anton\node_modules\@phenomnomnominal\tsquery\dist\src\ast.js
   - C:\source\test14\anton\node_modules\@phenomnomnominal\tsquery\dist\src\index.js
   - C:\source\test14\anton\node_modules\@nxtensions\astro\src\generators\application\utilities\configure-tailwind-integration.js
   - C:\source\test14\anton\node_modules\@nxtensions\astro\src\generators\application\utilities\index.js
   - C:\source\test14\anton\node_modules\@nxtensions\astro\src\generators\application\generator.js
   - C:\source\test14\anton\node_modules\@nxtensions\astro\src\generators\preset\generator.js
   - C:\source\test14\anton\node_modules\nx\src\config\schema-utils.js
   - C:\source\test14\anton\node_modules\nx\src\command-line\generate\generator-utils.js
   - C:\source\test14\anton\node_modules\nx\src\command-line\generate\generate.js
   - C:\source\test14\anton\node_modules\nx\src\command-line\generate\command-object.js
   - C:\source\test14\anton\node_modules\nx\src\command-line\nx-commands.js
   - C:\source\test14\anton\node_modules\nx\bin\init-local.js
   - C:\source\test14\anton\node_modules\nx\bin\nx.js
   Pass --verbose to see the stacktrace.


 >  NX   Failed to apply preset: @nxtensions/astro

   See above
```

### Link to minimal reproducible example

_No response_

### Environment

```
 >  NX   The current directory isn't part of an Nx workspace.

   To create a workspace run:
   npx create-nx-workspace@latest <workspace name>

   To add Nx to an existing workspace with a workspace-specific nx.json, run:
   npx nx@latest init


 >  NX   For more information please visit https://nx.dev/
 ```

--- a/packages/astro/src/generators/application/generator.ts
+++ b/packages/astro/src/generators/application/generator.ts
@@ -27,7 +27,7 @@ export async function applicationGenerator(
   addProject(tree, options);
   setDefaultProject(tree, options.projectName);
   addFiles(tree, options);
-  configureTailwindIntegration(tree, options);
+  await configureTailwindIntegration(tree, options);
 
   const e2eTask = await setupE2ETests(tree, options);
   if (e2eTask) {

--- a/packages/astro/src/generators/application/utilities/configure-tailwind-integration.ts
+++ b/packages/astro/src/generators/application/utilities/configure-tailwind-integration.ts
@@ -4,16 +4,15 @@ import {
   readJson,
   type Tree,
 } from '@nx/devkit';
-import { ast, query } from '@phenomnomnominal/tsquery';
 import { clean, coerce, lt } from 'semver';
 import type { CallExpression } from 'typescript';
 import { getInstalledNxVersion } from '../../utilities/versions';
 import type { IntegrationInfo, NormalizedGeneratorOptions } from '../schema';
 
-export function configureTailwindIntegration(
+export async function configureTailwindIntegration(
   tree: Tree,
   options: NormalizedGeneratorOptions
-): void {
+): Promise<void> {
   const tailwindIntegration = options.integrations.find(
     (integration) => integration.name === 'tailwind'
   );
@@ -34,6 +33,8 @@ export function configureTailwindIntegration(
     'astro.config.mjs'
   );
   const astroConfigContent = tree.read(astroConfigPath, 'utf-8');
+
+  const { ast, query } = await import('@phenomnomnominal/tsquery');
   let source = ast(astroConfigContent);
 
   const tailwindCallExpressions = query<CallExpression>(

